### ISSUE TITLE
Link to new upstream repo

### ISSUE THREAD

**Issue by Flamefire** [S25RTTR-Aux/s25rttr#10]
There is an official Git repo for RTTR and patches are very welcome: https://github.com/Return-To-The-Roots/s25client

Please feel free to link to this instead and/or remove this repo so efforts can be bundled and gains maximized :)

**Comment by nevik** [comment]
Hey, thanks for letting me know! :) 

This repository is no longer active, and doesn't really contain anything that's of use any more. So I'm closing it down. 

Cool thing that RTTR is now on github 😄

**Comment by Flamefire** [comment]
Yes and there have also been quite some changes since it moved there :)
If possible, you can try to backport your changes to the new repo, if it is worth the work.
